Makefile

```diff
@@ -133,43 +133,39 @@ build: generate fmt vet ## Build manager binary.
+	cd $(LOCALBIN) && mkdir -p default-config && cp ../config/manager/$(CONF_DIR)/* default-config && ./manager
 
+# by default images expire from quay registry after 14 days
+# set a longer timeout (or set no label to keep images forever)
+LABEL ?= quay.expires-after=14d
 PLATFORM ?= linux/amd64
 # If you wish built the manager image targeting other platforms you can use the --platform flag.
 # (i.e. docker build --platform linux/arm64 ). However, you must enable docker buildKit for it.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
-.PHONY: docker-build
-docker-build: test ## Build docker image with the manager using docker.
-	docker build --platform ${PLATFORM} -t ${IMG} . -f docker/Dockerfile
 
-.PHONY: docker-push
-docker-push: ## Push docker image with the manager using docker.
-	docker push ${IMG}
+.PHONY: operator-build
+operator-build: test ## Build docker image with the manager using docker.
+	$(CONTAINER_ENGINE) build --platform $(PLATFORM) -f docker/Dockerfile -t $(IMG) --label $(LABEL) .
 
-.PHONY: podman-build
-podman-build: test ## Build docker image with the manager using podman.
-	podman build --platform ${PLATFORM} -t ${IMG} . -f docker/Dockerfile
-
-.PHONY: podman-push
-podman-push: ## Push docker image with the manager using podman.
-	podman push ${IMG}
+.PHONY: operator-push
+operator-push: ## Push IMG image to registry
+	$(CONTAINER_ENGINE) push $(IMG)
 
 # PLATFORMS defines the target platforms for  the manager image be build to provide support to multiple
 # architectures. (i.e. make docker-buildx IMG=myregistry/mypoperator:0.0.1). To use this option you need to:
 # - able to use docker buildx . More info: https://docs.docker.com/build/buildx/
 # - have enable BuildKit, More info: https://docs.docker.com/develop/develop-images/build_enhancements/
-# - be able to push the image for your registry (i.e. if you do not inform a valid value via IMG=<myregistry/image:<tag>> than the export will fail)
+# - be able to push the image for your registry (i.e. if you do not inform a valid value via IMG=<myregistry/image:tag> than the export will fail)
 # To properly provided solutions that supports more than one platform you should use this option.
 # If more arches are needed, use comma-separated list: linux/amd64,linux/arm64,linux390x,linux/ppc64le
 PLATFORMS ?= linux/amd64
 .PHONY: docker-buildx
 docker-buildx: test ## Build and push docker image for the manager for cross-platform support
-	# copy existing docker/Dockerfile and insert --platform=${BUILDPLATFORM} into docker/Dockerfile.cross, and preserve the original Dockerfile
-	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' docker/Dockerfile > docker/Dockerfile.cross
+	# copy existing docker/Dockerfile and insert --platform=${PLATFORMS} into docker/Dockerfile.cross, and preserve the original Dockerfile
```

Comment:
I think `BUILDPLATFORM` is a special build argument that is automatically made available by Docker. See https://docs.docker.com/build/building/multi-platform/#cross-compilation
Otherwise, this step won't work:
```
$ make docker-buildx

test -s /home/asoro/work/projects/backstage/janus-idp/operator/bin/controller-gen || GOBIN=/home/asoro/work/projects/backstage/janus-idp/operator/bin go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.11.3
/home/asoro/work/projects/backstage/janus-idp/operator/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
/home/asoro/work/projects/backstage/janus-idp/operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
test -s /home/asoro/work/projects/backstage/janus-idp/operator/bin/goimports || GOBIN=/home/asoro/work/projects/backstage/janus-idp/operator/bin go install golang.org/x/tools/cmd/goimports@v0.15.0
find . -not -path '*/\.*' -name '*.go' -exec /home/asoro/work/projects/backstage/janus-idp/operator/bin/goimports -w {} \;
go vet ./...
test -s /home/asoro/work/projects/backstage/janus-idp/operator/bin/setup-envtest || GOBIN=/home/asoro/work/projects/backstage/janus-idp/operator/bin go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
mkdir -p /home/asoro/work/projects/backstage/janus-idp/operator/bin/default-config && cp config/manager/default-config/* /home/asoro/work/projects/backstage/janus-idp/operator/bin/default-config
LOCALBIN=/home/asoro/work/projects/backstage/janus-idp/operator/bin KUBEBUILDER_ASSETS="/home/asoro/work/projects/backstage/janus-idp/operator/bin/k8s/1.25.0-linux-amd64" go test ./... -coverprofile cover.out
?       janus-idp.io/backstage-operator [no test files]
?       janus-idp.io/backstage-operator/api/v1alpha1    [no test files]
ok      janus-idp.io/backstage-operator/controllers     33.044s coverage: 74.2% of statements
# copy existing docker/Dockerfile and insert --platform=linux/amd64,linux/arm64 into docker/Dockerfile.cross, and preserve the original Dockerfile
sed -e '1 s/\(^FROM\)/FROM --platform=\$\{PLATFORMS\}/; t' -e ' 1,// s//FROM --platform=\$\{PLATFORMS\}/' docker/Dockerfile > docker/Dockerfile.cross
docker buildx create --name project-v3-builder
project-v3-builder
docker buildx use project-v3-builder
docker buildx build --push --platform=linux/amd64,linux/arm64 --tag quay.io/asoro/backstage-operator . -f docker/Dockerfile.cross  --label quay.expires-after=14d
[+] Building 2.1s (2/2) FINISHED                                                                                                                           docker-container:project-v3-builder
 => [internal] booting buildkit                                                                                                                                                           1.9s
 => => pulling image moby/buildkit:buildx-stable-1                                                                                                                                        1.2s
 => => creating container buildx_buildkit_project-v3-builder0                                                                                                                             0.7s
 => [internal] load build definition from Dockerfile.cross                                                                                                                                0.1s
 => => transferring dockerfile: 2.59kB                                                                                                                                                    0.0s
Dockerfile.cross:16
--------------------
  14 |     
  15 |     #@follow_tag(registry.redhat.io/rhel9/go-toolset:latest)
  16 | >>> FROM --platform=${PLATFORMS} registry.access.redhat.com/ubi9/go-toolset:1.20.10-6 as builder
  17 |     # hadolint ignore=DL3002
  18 |     USER 0
--------------------
ERROR: failed to solve: failed to parse platform : "" is an invalid component of "": platform specifier component must match "^[A-Za-z0-9_-]+$": invalid argument
make: [Makefile:168: docker-buildx] Error 1 (ignored)
docker buildx rm project-v3-builder
project-v3-builder removed
```